src/shared/sam/sync.ts

```diff
@@ -96,17 +101,27 @@ function createStackPrompter(client: DefaultCloudFormationClient) {
 
     return createQuickPick(items, {
         title: 'Select a CloudFormation Stack',
-        placeholder: 'Filter or enter a new stack name',
-        filterBoxInputSettings: {
-            label: 'Create a New Stack',
-            transform: v => v,
+        placeholder: 'Select a stack',
+        // filterBoxInputSettings: {
+        //     label: 'Create a New Stack',
+        //     transform: v => v,
+        // },
```

Comment:
@JadenSimon  what is this doing? Is it ok to comment-out these pass-through `filterBoxInputSettings` fields for now? Seems like `noItemsFoundItem` does not work if they are present.

Reply:
I could never come up with a good name for `filterBoxInputSettings` but it's intent is to create an item from the user's input. So in this case the user can choose to create a new stack from their input if one isn't already available in the picker.